lib/units/device/resources/service.js

```diff
@@ -17,7 +17,7 @@ module.exports = syrup.serial()
       pathutil.vendor('STFService/wire.proto'))
 
     var resource = {
-      requiredVersion: '2.4.6'
+      requiredVersion: '2.4.7'
```

Comment:
There is no such version on https://github.com/DeviceFarmer/STFService.apk repo.
Is it intended to use it here now?

Reply:
That's the latest addition from https://github.com/DeviceFarmer/STFService.apk/pull/42
https://github.com/DeviceFarmer/STFService.apk/blob/b75975682a5e4d32c37881515dfc07b3467221d6/app/build.gradle#L29
I think we can release it now.